test/Dapr.AspNetCore.IntegrationTest/SubscribeEndpointTest.cs

```diff
@@ -32,14 +32,20 @@ public async Task SubscribeEndpoint_ReportsTopics()
                     json.ValueKind.Should().Be(JsonValueKind.Array);
                     json.GetArrayLength().Should().Be(3);
                     var topics = new List<string>();
+                    var routes = new List<string>();
                     foreach (var element in json.EnumerateArray())
                     {
-                        topics.Add(element.GetString());
+                        topics.Add(element.GetProperty("topic").GetString());
+                        routes.Add(element.GetProperty("route").GetString());
                     }
 
                     topics.Should().Contain("A");
                     topics.Should().Contain("B");
                     topics.Should().Contain("register-user");
+
+                    routes.Should().Contain("B");
+                    routes.Should().Contain("/topic-a");
+                    routes.Should().Contain("register-user");
```

Comment:
I assume the test code already has a case where topic name is set to something different from the route?

Reply:
Yes, it had different routes.